cnj/src/main.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import os
+import datetime
+import crawler
+import parser
+
+if('COURT' in os.environ):
+    court = os.environ['COURT']
+else:
+    sys.stderr.write("Invalid arguments, missing parameter: 'COURT'.\n")
+    os._exit(1)
+if('DRIVER_PATH' in os.environ):
+    driver_path = os.environ['DRIVER_PATH']
+else:
+    sys.stderr.write("Invalid arguments, missing parameter: 'DRIVER_PATH'.\n")
+    os._exit(1)
+if('OUTPUT_FOLDER' in os.environ):
+    output_path = os.environ['OUTPUT_FOLDER']
+else:
+    output_path = "/output"
+if('GIT_COMMIT' in os.environ):
+    crawler_version = os.environ['GIT_COMMIT']
+else:
+    sys.stderr.write("crawler_version cannot be empty")
+    os._exit(1)
+
+# Main execution
+if __name__ == '__main__':
+    file_names = crawler.crawl(court, driver_path, output_path)
+    
+    now = datetime.datetime.now()
+    current_month = now.month
+    current_year = now.year
+    months = list(range(1, 13))
+    years = list(range(2018, current_year + 1))
+
+    for year in years:
+        for month in months:
+            if year == current_year and month >= current_month:
+                break
+            if month < 10:
+                parsing_date = "0" + str(month) + "/" + str(year)
+            else:
+                parsing_date = str(month) + "/" + str(year)
+
```

Comment:
Essa lógica de parsing_date existe por causa do filtro no data frame. Dessa forma, essa conversão para string deve estar mais perto de lá (pelo princício de projeto expert).

Dessa forma, sugiro que esse código seja movido mais para perto de onde o data frame é filtrado e os parâmetros mês e ano sejam passados do `parse` até lá.

Note que não estou sugerindo que mês e ano virem parâmetros do coletor. Está bacana dentro do for e etc. Devem ser parâmetros do método `parse`